test/test_image.py

```diff
@@ -48,3 +49,27 @@ def test_gradcheck(self):
         input = utils.tensor_to_gradcheck_var(input)  # to var
         assert gradcheck(image.gaussian_blur, (input, kernel_size, sigma,),
                          raise_exception=True)
+
+
+@pytest.mark.parametrize("window_size", [5])
+def test_get_laplacian_kernel(window_size):
+    kernel = image.get_laplacian_kernel(window_size)
+    assert kernel.shape == (window_size,)
+    assert kernel.sum().item() == pytest.approx(1.0)
+
+
+def test_get_laplacian_kernel2d(window_size):
+    kernel = image.get_laplacian_kernel2d(window_size)
+    assert kernel.shape == window_size
+    assert kernel.sum().item() == pytest.approx(1.0)
+
+
+class TestLaplacianBlur:
+    @pytest.mark.parametrize("batch_shape",
+                             [(1, 4, 8, 15), (2, 3, 11, 7)])
+    def test_laplacian_blur(self, batch_shape, device_type):
+        kernel_size = 5
+
+        input = torch.rand(batch_shape).to(torch.device(device_type))
+        laplace = image.LaplacianBlur(kernel_size)
```

Comment:
add a test and use opencv to compute the laplacian for a given input to see if we match

Reply:
Let me know after you verify which implementation of Laplacian do we want? Then, I can add this test.

Reply:
I don't see why opencv uses sobel filter for laplacian, but after a bit of research I think that in the way we are doing should be fine. @bhack do you have any insight why is that ?

I've found this interesting test: https://stackoverflow.com/questions/32768407/what-is-the-laplacian-mask-kernel-used-in-the-scipy-ndimage-filter-laplace

Reply:
I did some more digging.
Refer the link:
https://github.com/scipy/scipy/blob/v0.16.0/scipy/ndimage/filters.py#L396

Please, run the following code snippet (to verify) which I wrote to see what scipy spits out:
``
from scipy.ndimage import *
import numpy as np

A = np.zeros((7,7))
A[3,3] = 1
print(A)

#I have extended the filter code in scipy to account for any kernel size [1, 1, -4, 1, 1] kernel size=5 here:

def derivative2(input, axis, output, mode, cval):
        return correlate1d(input, [1, 1, -4, 1, 1], axis, output, mode, cval, 0)

B = generic_laplace(A, derivative2)
print(B)

#And this is what scipy gives because it uses [1, -2, 1] as the only kernel size of 3
C= scipy.ndimage.filters.laplace(A)

print(C)
``
A = array([[0., 0., 0., 0., 0., 0., 0.],
       [0., 0., 0., 0., 0., 0., 0.],
       [0., 0., 0., 0., 0., 0., 0.],
       [0., 0., 0., 1., 0., 0., 0.],
       [0., 0., 0., 0., 0., 0., 0.],
       [0., 0., 0., 0., 0., 0., 0.],
       [0., 0., 0., 0., 0., 0., 0.]])

B = array([[ 0.,  0.,  0.,  0.,  0.,  0.,  0.],
       [ 0.,  0.,  0.,  1.,  0.,  0.,  0.],
       [ 0.,  0.,  0.,  1.,  0.,  0.,  0.],
       [ 0.,  1.,  1., -8.,  1.,  1.,  0.],
       [ 0.,  0.,  0.,  1.,  0.,  0.,  0.],
       [ 0.,  0.,  0.,  1.,  0.,  0.,  0.],
       [ 0.,  0.,  0.,  0.,  0.,  0.,  0.]])

C = array([[ 0.,  0.,  0.,  0.,  0.,  0.,  0.],
       [ 0.,  0.,  0.,  0.,  0.,  0.,  0.],
       [ 0.,  0.,  0.,  1.,  0.,  0.,  0.],
       [ 0.,  0.,  1., -4.,  1.,  0.,  0.],
       [ 0.,  0.,  0.,  1.,  0.,  0.,  0.],
       [ 0.,  0.,  0.,  0.,  0.,  0.,  0.],
       [ 0.,  0.,  0.,  0.,  0.,  0.,  0.]])

Now, one would ask is why B slightly different from what I have in PyTorch geometry?
Because, I consider cross-derivatives as well. See here:

D =  [1., 1., 1., 1., 1.],
        [1., 1., 1., 1., 1.],
        [1., 1., -24., 1, 1.],
        [1., 1., 1., 1., 1.],
        [1., 1., 1., 1., 1.],

Reply:
Hope this now clears the differences between scipy, OpenCV and my implementation.